packages/graphql/src/schemaTypes/objectTypes/gql-CurrentProject.ts

```diff
@@ -75,9 +74,9 @@ export const CurrentProject = objectType({
 
     t.connectionField('specs', {
       description: 'Specs for a project conforming to Relay Connection specification',
-      type: Spec,
-      nodes: (source, args, ctx) => {
-        return ctx.project.findSpecs(source.projectRoot, ctx.appData.currentTestingType === 'component' ? 'component' : 'integration')
+      type: 'Spec',
+      nodes: async (source, args, ctx) => {
```

Comment:
`async` is not necessary

Reply:
https://github.com/cypress-io/cypress/pull/19319/commits/9a13720f8d4c881a8edac8408c9239a353fc8ac5